src/main/kotlin/no/skatteetaten/aurora/boober/service/internal/DeploymentConfigGenerator.kt

```diff
@@ -45,10 +46,21 @@ class DeploymentConfigGenerator(
             auroraContainer {
 
                 name = "${auroraDeploymentSpec.name}-${adcContainer.name}"
-                ports = adcContainer.ports
+                ports = adcContainer.tcpPorts.map {
```

Comment:
In this PR the kubernetes objects are created in this class/layer and not in any of the layers before it.